apps/backend/src/main/resources/application.yml

```diff
@@ -23,3 +23,7 @@ spring:
       lifecycle-management: start-only
 springdoc:
   pathsToMatch: /api/**
+security:
+  jwt:
+    secret-key: 909912191940e298a1157bfdebb1f5f785daf15b93a1787b80350deab6cdce55
```

Comment:
Könnte man das noch als GitHub Env Variable definieren? Gibt es eine Möglichkeit Dev Config und Prod Config zu aufzuteile?

Reply:
Yes, guete Punkt - has aktuell no im TODO.md als "offne Punkt".
Note to Self: Nöis Secret generiere, das hie isch ir Git History ersichtlech

Reply:
Siehe Teams: Chunnt imne eigete PR